### PR TITLE
feat(react): Fix the React dev development link

### PR DESCRIPTION
**Pull request by pe-3**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [X] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

Fix the React dev development link.

### CODE REVIEW

File: internals/cli/src/commands/build/build-ui.ts [hunk 1]
```diff
@@ -242,62 +242,62 @@ async function batchBuildAll({ vueVersion, tasks, formats, message, emptyOutDir,
...
+    //     },
+    //     lib: {
+    //       // 这里可以多入口打包，也可以单入口打包
+    //       entry,
+    //       formats,
+    //       fileName: (format, entryName) => `${entryName}.js`
+    //     },
+    //     outDir
+    //   }
+    // })
```
Comment (GaoNeng-wWw) [inline internals/cli/src/commands/build/build-ui.ts]:
```suggestion
```

删除比注释更好

File: examples/react-docs/src/main.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+.app {
+  margin: 10px;
+  width: 500px;
+}
```
Comment (GaoNeng-wWw) [inline examples/react-docs/src/main.css:8]: 用了tailwind，是不是直接写className更好？

File: internals/cli/src/commands/build/build-ui-react.ts [hunk 3]
```diff
@@ -405,6 +405,8 @@ async function batchBuild({
   logGreen(`====== 开始构建 ${message} ======`)
   const entry = toEntry(tasks)
   const dtsInclude = toTsInclued(tasks)
+  console.log(entry)
+
```
Comment (GaoNeng-wWw) [inline internals/cli/src/commands/build/build-ui-react.ts]:
```suggestion
```

File: internals/cli/src/commands/build/build-ui.ts [hunk 4]
```diff
@@ -242,62 +242,62 @@ async function batchBuildAll({ vueVersion, tasks, formats, message, emptyOutDir,
     if (tasks.length === 0) return
     logGreen(`====== 开始构建 ${message} ======`)
     const entry = toEntry(tasks)
-
+    console.log(entry)
```
Comment (GaoNeng-wWw) [inline internals/cli/src/commands/build/build-ui.ts]:
```suggestion
```

File: packages/react-common/src/vue-hooks.js [hunk 5]
```diff
@@ -10,11 +10,11 @@ function Readonly(target) {
   Create.call(this, target)
 }
 
-const useDepChange = (dependencies) => {
+const useDepChange = (dependencies, immediate = false) => {
   let isDepChange = false
   const pre_dep = useRef()
   if (!pre_dep.current) {
-    isDepChange = true
+    isDepChange = true && immediate
```
Comment (GaoNeng-wWw) [inline packages/react-common/src/vue-hooks.js:17]: 这里的逻辑是不是冗余了

File: pnpm-lock.yaml
Comment (GaoNeng-wWw) [inline pnpm-lock.yaml:1]: 这个文件不要提交

**Review by GaoNeng-wWw — CHANGES_REQUESTED** [review]

File: examples/react-docs/package.json [hunk 6]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@opentiny/react-docs",
+  "name": "@pe-3/react-docs",
```
Comment (kagol) [inline examples/react-docs/package.json]: 别改这个

**Comment by kagol** [comment]
@pe-3 不要提交 lock 文件

File: examples/react-docs/src/main.css (hunk 2 above)
Comment (pe-3) [reply examples/react-docs/src/main.css:8]: 好的，大哥

File: packages/react-common/src/vue-hooks.js (hunk 5 above)
Comment (pe-3) [reply packages/react-common/src/vue-hooks.js:17]: 我都参考一下

File: examples/react-docs/package.json (hunk 6 above)
Comment (pe-3) [reply examples/react-docs/package.json]: 好的

**Comment by pe-3** [comment]
> @pe-3 不要提交 lock 文件

好的

**Review by kagol — APPROVED** [review]

**Pull request merged**